apps/src/music/views/Callouts.tsx

```diff
@@ -8,9 +8,9 @@ import {MusicState} from '../redux/musicRedux';
 
 import moduleStyles from './callouts.module.scss';
 
-const arrowImage = require(`@cdo/static/music/music-callout-arrow.png`);
+const arrowImage = require(`@cdo/static/music/music-callout-arrow-outline.png`);
 
-type DirectionString = 'up' | 'left';
+type DirectionString = 'up' | 'left' | 'up-inside' | 'up-left';
```

Comment:
What is `up-inside` as compared to `up`? Does it just have more vertical space above?

Reply:
`up` points up, at the bottom of the block (and at the horizontal center of the block).  

`up-inside` points up, but at the the "inside" of a C-shaped block, determined by a hardcoded offset from the top-left corner of the block.  Here, it's particularly useful as the destination for a moving arrow.